ShadowverseTracker/ShadowverseTracker.xcodeproj/project.pbxproj

```diff
@@ -260,6 +272,7 @@
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```

Comment:
@Ruann this is the part that worries me, should this be added to the project file?

Reply:
I think is ok to leave as is.